### DISCUSSION TITLE
How to set the sampling time interval?

### DISCUSSION THREAD

**Discussion by lk1983823** [johbrust/simulink_gym#8] [Q&A] [answered]
I can successfully run the cartpole_simulink.ipynb. However, I find the sampling interval is 0.02s by checking info['simulation time [s]']. Does the interval depend on the tcp/ip sending frequency or be possible to set manually ? How can set another interval, like 1s. Thank you.

**Comment by johbrust** [comment, marked as answer]
Since you already closed the issue I conclude that you solved it yourself. But just in case and for any future user with this issue:

For the cart pole example, the step size is an [attribute of the object initialization](https://github.com/johbrust/simulink_gym/blob/main/examples/envs/cartpole_simulink/cartpole_simulink.py#L41).
